### ISSUE TITLE
paren can wrap by itself if after a link

### ISSUE THREAD

**Issue by drifkin** [lumen-notes/lumen#254]
just a tiny little (and probably extremely rare) typographical thing that I happened to run into today that looked a little odd:

![Image](https://github.com/lumen-notes/lumen/assets/175530/a58de4ad-129d-447b-91ac-56507aa6a6af)

also while reproing it, I noticed that the link indicator arrow thingy also will wrap, which probably should stick with the url instead
![Image](https://github.com/lumen-notes/lumen/assets/175530/77ffa485-4c26-4463-9d8e-08001dc9e4eb)

super nitpicky 😁

**Comment by colebemis** [comment]
Nice catch! This is a tricky one. What would you expect to happen in this case? Should "link" wrap to the second line?

**Comment by colebemis** [comment]
I also don't love when punctuation (like `)` or `.`) wraps to a line by itself.

**Comment by drifkin** [comment]
> Nice catch! This is a tricky one. What would you expect to happen in this case? Should "link" wrap to the second line?

yeah, exactly. I think that's how plaintext would normally wrap, though until this I never actually thought about it
